module/utils.mjs

```diff
@@ -49,10 +49,12 @@ export function formatModifier(mod) {
  * @param {boolean} [options.numerals]  Format the number as roman numerals.
  * @param {boolean} [options.ordinal]   Use ordinal formatting.
  * @param {boolean} [options.words]     Write out number as full word, if possible.
+ * @param {string} [options.blank]      Format a zero or otherwise empty value as the given string.
```

Comment:
```suggestion
 * @param {string} [options.blank]      Format a zero or otherwise empty value as the given string.
 * @param {boolean} [options.numerals]  Format the number as roman numerals.
 * @param {boolean} [options.ordinal]   Use ordinal formatting.
 * @param {boolean} [options.words]     Write out number as full word, if possible.
```